core/blockchain/blockchain_utils.py

```diff
@@ -33,7 +33,7 @@ def download(client: object, key: str, state: list) -> list:
     relevant_txs = list(
         map(lambda tx: 
                 ipfs_to_content(client, tx.get(TxEnum.CONTENT.name)),
-        filter(lambda tx: 
+                filter(lambda tx: 
                 tx.get(TxEnum.KEY.name) == key, state)))
```

Comment:
If this fits on the previous line then add it there; otherwise indent

Reply:
Done